frontend/app/components/contextual-alert.tsx

```diff
@@ -1,26 +1,44 @@
 import type { ReactNode } from 'react';
 
+import { faCommentDots } from '@fortawesome/free-regular-svg-icons';
 import { faCheckCircle, faCircleInfo, faExclamationCircle, faExclamationTriangle } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 import { cn } from '~/utils/tw-utils';
 
+export type AlertType = 'warning' | 'success' | 'danger' | 'info' | 'comment' | 'default';
+
 export interface ContextualAlertProps {
   children: ReactNode;
-  type: 'warning' | 'success' | 'danger' | 'info';
+  type: AlertType;
 }
 
+const alertBackgroundColors: Partial<Record<AlertType, string>> = {
```

Comment:
```suggestion
const alertBackgroundColors: Partial<Record<AlertType, string>> & { default: string } = {
```

Reply:
this doesn't work, but `Partial<Record<AlertType | 'default', string>` does.

Reply:
Remove ` | 'default'` as `AlertType`.

Reply:
?? I get a compile error otherwise.

Reply:
Works for me. Look at this [commit](https://github.com/DTS-STN/canadian-dental-care-plan/commit/29392fea37eb0aa8b2848ac6bb7c5fe49b65697b).

Reply:
:shrug:  I don't know what happened, but you're right.  It works